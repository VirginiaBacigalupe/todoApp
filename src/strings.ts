import LocalizedStrings from 'react-native-localization'

export const strings = new LocalizedStrings({
  'en-US': {
    cancelButton: 'Cancel',
    clearAllButton: 'CLEAR ALL DONE',
    done: 'Done',
    edit: 'Edit',
    editTaskHeader: 'Edit Task',
    emptyListText: 'The tasks list is empty',
    initialHeaderTitle: 'Todo',
    invalidDescriptionError: 'Please enter a description for the task.',
    invalidTitleAndDescriptionError:
      'Please enter a title and a description for the task.',
    invalidTitleError: 'Please enter a title for the task.',
    markButton: 'MARK AS DONE',
    newTaskHeader: 'New Task',
    notDone: 'Not done',
    saveButton: 'Save',
    unMarkButton: 'UNMARK',
  },
})
