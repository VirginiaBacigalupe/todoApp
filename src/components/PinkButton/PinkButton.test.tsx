import { shallow } from 'enzyme'
import React from 'react'

import { strings } from '../../strings'
import { PinkButton } from './index'

describe('PinkButton', () => {
  it('Pink button renders ok', () => {
    const onPressMock = jest.fn()
    const title = strings.clearAllButton
    const wrapper = shallow(<PinkButton title={title} onPress={onPressMock} />)
    expect(wrapper).toMatchSnapshot()
  })
})
