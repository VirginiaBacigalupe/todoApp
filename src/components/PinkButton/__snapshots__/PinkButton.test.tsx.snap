// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PinkButton Pink button renders ok 1`] = `
<ForwardRef
  onPress={[MockFunction]}
  style={
    Object {
      "alignItems": "center",
    }
  }
>
  <Text
    style={
      Object {
        "alignItems": "center",
        "color": "rgb(255, 25, 123)",
        "fontWeight": "bold",
        "margin": 20,
      }
    }
  >
    CLEAR ALL DONE
  </Text>
</ForwardRef>
`;
